packages/oas-form/src/components/fields/StringField.js

```diff
@@ -27,8 +27,20 @@ function StringField(props) {
   if (format && hasWidget(schema, format, widgets)) {
     defaultWidget = format;
   }
-  const { widget = defaultWidget, placeholder = '', ...options } = getUiOptions(uiSchema);
+  const { widget = defaultWidget, ...options } = getUiOptions(uiSchema);
   const Widget = getWidget(schema, widget, widgets);
+
+  // If we have examples present, let's pull the first out as a placeholder if it's a primitive value.
+  let placeholder = '';
```

Comment:
so much work and prs to support these 9 lines of code. definitely worth it to end up with something so simple and intuitive in the stringfield library.